v2/support/instances_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,34 @@
+<html>
+<body>
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    var client = ZAFClient.init();
+
+    function commentTemplate(currentUser, requester, lineBreak) {
+      return 'Hi ' + currentUser + ',' + lineBreak + lineBreak + 'Kind regards,' + lineBreak + requester;
+    }
+
+    client.on('instance.created', function(context) {
+      if (context.location === 'ticket_sidebar') {
+        var ticketInstance = client.instance(context.instanceGuid);
+
+        ticketInstance.get([
+          'currentUser.name',
```

Comment:
I think that will complicate things a bit. Will have to have a separate promise, then have an `all().then` call here. I'd rather keep it simple as an example.